From 7f7f1b19155773ae3cd43a004bab3a50d6e2398d Mon Sep 17 00:00:00 2001
From: Rex Zhu <Rex.Zhu@amd.com>
Date: Mon, 6 Aug 2018 19:45:04 +0800
Subject: [PATCH 0031/2940] drm/amdgpu: Delay 100ms to enable gfx off feature

Original 500ms delay seems a bit large.
Change to 100 ms suggested by Christian.

Signed-off-by: Rex Zhu <Rex.Zhu@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_gfx.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_gfx.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_gfx.c
index bbb81e23020e..790fd5408ddf 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_gfx.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_gfx.c
@@ -26,8 +26,8 @@
 #include "amdgpu.h"
 #include "amdgpu_gfx.h"
 
-/* 0.5 second timeout */
-#define GFX_OFF_DELAY_ENABLE         msecs_to_jiffies(500)
+/* delay 0.1 second to enable gfx off feature */
+#define GFX_OFF_DELAY_ENABLE         msecs_to_jiffies(100)
 
 /*
  * GPU GFX IP block helpers function.
-- 
2.17.1

